code/src/reducers/Tasks.js

```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const tasks = createSlice({
+  name: 'tasks',
+  initialState: {
+    items: [
+      { id: 1, todo: 'Wake up 👌', checkedTask: false},
```

Comment:
Nice to have already completed a task when opening the app 👍

Reply:
This made me smile!

Reply:
Maybe add a manual `createdAt` here (otherwise it will say the time you load the page)